Kconfig.nrf

```diff
@@ -12,6 +12,21 @@ orsource "./$(BOARD_DIR)/Kconfig"
 
 rsource "boards/shields/*/Kconfig.shield"
 rsource "subsys/net/openthread/Kconfig.defconfig"
+rsource "modules/lib/matter/config/nrfconnect/chip-module/Kconfig.defconfig"
```

Comment:
I'm not sure about the idea behind this change, but I don't think it's necessary to move these configs for the Matter project. First of all we source Kconfig for these defaults even before Kconfig.zephyr, so I think it will be always sourced before nrf_security configs. 
Apart from that, we handle crypto defaults in Matter repository fork to be consistent with the Matter upstream repo. We also source the Kconfig.defaults only in Matter samples' Kconfig files, as an example configuration for the user, but we do not want to source it always, when building Matter library.

Reply:
Aha, so it is every samples responsibility to source modules/lib/matter/config/nrfconnect/chip-module/Kconfig.defaults ?

Reply:
Does your comment apply to the line below as well?

Reply:
reverted, hopefully CI will be green with the revert :)

Reply:
> Does your comment apply to the line below as well?

Yes.